common/djangoapps/course_modes/models.py

```diff
@@ -100,8 +101,17 @@ class CourseMode(models.Model):
     NO_ID_PROFESSIONAL_MODE = "no-id-professional"
     CREDIT_MODE = "credit"
 
-    DEFAULT_MODE = Mode(HONOR, _('Honor'), 0, '', 'usd', None, None, None)
-    DEFAULT_MODE_SLUG = HONOR
+    DEFAULT_MODE = Mode(
```

Comment:
It may help if there was a comment here that we're doing this because upstream have 'audit' as their default course mode.